schema/partner_show.js

```diff
@@ -100,6 +101,14 @@ const PartnerShowType = new GraphQLObjectType({
       type: new GraphQLList(Artist.type),
       resolve: ({ artists }) => artists,
     },
+    galaxy_partner: {
```

Comment:
just make a new node for a galaxy partner. The CV/exhibition history fragment would then look like:

```
partner {
  name
}
galaxy_partner {
  name
}
```

I can adjust the template (or add a helper) to pick the exist-y one.